File: src/main/java/com/tpt/api_mbds/Controller/PariUserName.java
package com.tpt.api_mbds.Controller;

import java.util.Date;

public class PariUserName {
    private int id;
    private int idUtilisateur;
    private String nom;
    private int mise;
    private Date datepari;
    private String idMatch;
    private String matchRegle;

    public PariUserName() {
    }

    public PariUserName(int id, int idUtilisateur, String nom, int mise, Date datepari, String idMatch, String matchRegle) {
        this.id = id;
        this.idUtilisateur = idUtilisateur;
        this.nom = nom;
        this.mise = mise;
        this.datepari = datepari;
        this.idMatch = idMatch;
        this.matchRegle = matchRegle;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getIdUtilisateur() {
        return idUtilisateur;
    }

    public void setIdUtilisateur(int idUtilisateur) {
        this.idUtilisateur = idUtilisateur;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public int getMise() {
        return mise;
    }

    public void setMise(int mise) {
        this.mise = mise;
    }

    public Date getDatepari() {
        return datepari;
    }

    public void setDatepari(Date datepari) {
        this.datepari = datepari;
    }

    public String getIdMatch() {
        return idMatch;
    }

    public void setIdMatch(String idMatch) {
        this.idMatch = idMatch;
    }

    public String getMatchRegle() {
        return matchRegle;
    }

    public void setMatchRegle(String matchRegle) {
        this.matchRegle = matchRegle;
    }
}

File: winform/projetParis/projetParis/Model/PariUserName.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace projetParis.Model
{
    class PariUserName
    {
        private int id;
        private int idUtilisateur;
        private string nom;
        private int mise;
        private DateTime datepari;
        private string idMatch;
        private string matchRegle;

        public PariUserName()
        {
        }

        public PariUserName(int id, int idUtilisateur, string nom, int mise, DateTime datepari, string idMatch, string matchRegle)
        {
            Id = id;
            IdUtilisateur = idUtilisateur;
            Nom = nom;
            Mise = mise;
            Datepari = datepari;
            IdMatch = idMatch;
            MatchRegle = matchRegle;
        }

        public int Id { get => id; set => id = value; }
        public int IdUtilisateur { get => idUtilisateur; set => idUtilisateur = value; }
        public string Nom { get => nom; set => nom = value; }
        public int Mise { get => mise; set => mise = value; }
        public DateTime Datepari { get => datepari; set => datepari = value; }
        public string IdMatch { get => idMatch; set => idMatch = value; }
        public string MatchRegle { get => matchRegle; set => matchRegle = value; }
    }
}
